### PR TITLE
add a contributing.md

### PR DESCRIPTION
**Pull request by alicebartlett**
o-icons has some technical and design constrainst that are undocumented up to
this point. This commit adds a contributing.md, which documents how to contribute
to o-icons for developers in the origami team and outside of it.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# How to contribute to o-icons
+
+Thank you for your interest in contributing to o-icons! As our icons are used by a lot of different applications, we have a few constraints for new ones. If you aren't sure if your new icon meets any of these, please raise an issue](http://github.com/financial-times/o-icons/issues), or ask in the #ft-origami slack channel.
```
Comment (AlbertoElias) [inline contributing.md]: missing [ in issue

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+If your icon meets the design and technical criteria please follow the following steps and then open a Pull Request:
+
+1. Clone the repository and install dependencies:
+
+		git clone https://github.com/Financial-Times/o-icons.git
+		cd o-icons
+		obt install
+
+1. Add or edit an SVG file to the `svg` folder.
+1. Rebuild the demos so people can see the new icon over on http://registry.origami.com/components/o-icons. To do this, to run:
```
Comment (AlbertoElias) [inline contributing.md]: _to run_

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		obt demo --runServer
+
+
+## Removing an icon
+
+A lot of people use o-icons in many different ways. To remove an icon completely from o-icons, please [raise an issue](http://github.com/financial-times/o-icons/issues) so that the Origami team can manage the deprecation process.
+
+## How to test an icon with the Image Service
+
+The Image Service provides the ability to convert and SVG to a PNG. This is useful as a fallback for browsers that don't support SVGs.
```
Comment (AlbertoElias) [inline contributing.md]: and => an

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+The Image Service doesn't support clipPaths, so any SVG that has a clipPath in it may not render correctly. To check if your SVG works, upload it somewhere (eg dropbox, or gh-pages) and then run its public url through thes Image Service with `format=png` as a parameter. ie:
```
Comment (AlbertoElias) [inline contributing.md]: thes => the

**Comment by AlbertoElias** [comment]
Found a few typos, rest looks great 👍

**Comment by alicebartlett** [comment]
Thanks @AlbertoElias I've fixed those.

**Comment by AlbertoElias** [comment]
Feel free to :shipit:

**Pull request merged**